pyproject.toml

```diff
@@ -39,17 +39,17 @@ dependencies = [
     "httpx>=0.25.0",
     "importlib-metadata>=4.8.3;python_version<\"3.10\"",
     "importlib-resources>=1.4;python_version<\"3.9\"",
-    "ipykernel",
+    "ipykernel>=6.5.0",
     "jinja2>=3.0.3",
     "jupyter_core",
     "jupyter_server>=2.4.0,<3",
     "jupyter-lsp>=2.0.0",
     "jupyterlab_server>=2.19.0,<3",
     "notebook_shim>=0.2",
     "packaging",
-    "tomli;python_version<\"3.11\"",
+    "tomli>=1.2.2;python_version<\"3.11\"",
```

Comment:
This is carried over from traitlets pin, mostly needed for the resolution to pass see https://github.com/jupyterlab/maintainer-tools/issues/230